### PR TITLE
Added create product

### PR DESCRIPTION
**Pull request by dumitru-stratulat**

### CODE REVIEW

File: src/api/profile.tsx [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+  formData.append('price', price);
+  fileList.forEach(element => {
+    formData.append('image[]', element.originFileObj);
+  });
+
+  const response = await axios.post('https://reactive.loca.lt/create',
+    formData,
+    {
+      headers: {
+        Authorization: 'Bearer eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJlbWFpbCI6ImRpbWEuc3RyYXR1bGF0OTkxQGdtYWlsLmNvbSIsInVzZXJJZCI6IjVmNGZhMDIxNmVjZTYwZDA1ZTFjYWYyZiIsImlhdCI6MTU5OTEzODYzNiwiZXhwIjoxNTk5MzExNDM2fQ.HX1Yuw4LTPZSRr7iWgG5xdgIWiCnlD482grZynwZEmY'
```
Comment (AleVul) [inline src/api/profile.tsx:22]: suggest to keep session token in either cookie or local storage

File: src/components/FooterLayout/FooterLayout.tsx [hunk 2]
```diff
@@ -6,7 +6,7 @@ const FooterLayout: React.FC = () => {
   return <footer>
     <nav className={style.footer}>
       <div className={style.logo}>
-        <img src="#" alt="footer logo" />
+        {/* <img src="#" alt="footer logo" /> */}
```
Comment (AleVul) [inline src/components/FooterLayout/FooterLayout.tsx]: please remove commented line

File: src/pages/create.tsx [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+      >
+        <Form.Item label="Input" name="title">
+          <Input />
+        </Form.Item>
+        <Form.Item label="Description" name="content">
+          <Input.TextArea />
+        </Form.Item>
+        <Form.Item label="Categories" name="category">
+          <Cascader
+            options={[
```
Comment (AleVul) [inline src/pages/create.tsx]: suggest to move options prop value outside of the comment. Currently, at each rerender a new `options` array is passed which is not a good idea

Comment (dumitru-stratulat) [reply src/pages/create.tsx]: > suggest to move options prop value outside of the comment. 

Did you mean outside of the component?

Comment (AleVul) [reply src/pages/create.tsx]: yes

**Pull request merged**